packages/react/src/components/Slider/Slider.stories.js

```diff
@@ -53,6 +53,41 @@ export const SliderWithHiddenInputs = () => (
   />
 );
 
+export const SliderWithHiddenInputsAndFormatLabel = () => (
+  <Slider
+    labelText="Slider label with percentage"
+    value={50}
+    min={0}
+    max={100}
+    stepMultiplier={10}
+    step={1}
+    noValidate
+    hideTextInput
+    formatLabel={(val) => `${val}%`}
+  />
+);
+
+export const SliderWithCustomValueLabel = () => (
+  <Slider
+    labelText="Slider label with low/medium/high"
+    value={50}
+    min={0}
+    max={100}
+    stepMultiplier={10}
```

Comment:
We can change here to 50 so when the user press Shift + ArrowKey it increases to the next value.
```suggestion
    stepMultiplier={50}
```